src/css/popup.css

```diff
@@ -695,9 +695,8 @@ sign-up-panel::after {
   font-size: var(--fontSizeBodyXs);
   display: flex;
   justify-content: center;
-  gap: var(--spacingSm);
+  gap: var(--spacingLg);
   margin-bottom: var(--spacingSm);
-  align-items: center;
```

Comment:
Judging by the screenshot, doesn't removing this cause "Help us improve Relay" to be positioned slightly downwards, as opposed to in the middle of the message bar?

Reply:
<img width="180" alt="image" src="https://github.com/mozilla/fx-private-relay-add-on/assets/3924990/1206b126-63ef-4d4f-940b-59029c33b323">